habitat_sim/utils/data/data_extractor.py

```diff
@@ -99,10 +141,22 @@ def __getitem__(self, idx):
             out_name: obs[self.out_name_to_sensor_name[out_name]]
             for out_name in self.output
         }
-        sample["label"] = self.label_map[label]
+        sample["label"] = label
+        self.cache.add(cache_entry, sample)
 
         return sample
 
+    def precomute_tdv_and_refs(self, sim, scene_filepaths, pixels_per_meter):
```

Comment:
what is `tdv`? please add docstring. Changes to `data_extractor.py ` can be separate PR.